src/dist/replication/lib/mutation_log.h

```diff
@@ -468,7 +468,7 @@ class mutation_log_shared : public mutation_log
     // bufferring - only one concurrent write is allowed
     mutable zlock _slock;
     std::atomic_bool _is_writing;
-    std::shared_ptr<log_block> _pending_write;
+    std::unique_ptr<log_appender> _pending_write;
```

Comment:
Maybe `_pending_writes` or `_pending_blocks` is more suitable variety name？I just suggest this variety is plural form.

Reply:
Yes. But I want to keep changes as few as possible. On the other hand, `_pending_writes` is still understandable in the context. Since the blocks actually contain many "writes".